Bottom: 4c2b9923a7e15bf2ec42d9671f62f004940f435b
Top:    c41994545591f7423d6883e2775d83335af07532
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-13 20:10:26 +0000

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index 946668f..d2831a3 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1175,6 +1175,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1193,6 +1194,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
